File: TransitoAplicacionWeb/Transito/Models/Usuario.cs
using System;
using System.Collections.Generic;

namespace Transito.Models
{
    public partial class Usuario
    {
        public Usuario()
        {
            Dictamen = new HashSet<Dictamen>();
            UsuarioBitacoraAcceso = new HashSet<UsuarioBitacoraAcceso>();
        }

        public int Idusuario { get; set; }
        public string Nombre { get; set; }
        public string ApellidoPaterno { get; set; }
        public string ApellidoMaterno { get; set; }
        public int Idcargo { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }

        public Cargo IdcargoNavigation { get; set; }
        public ICollection<Dictamen> Dictamen { get; set; }
        public ICollection<UsuarioBitacoraAcceso> UsuarioBitacoraAcceso { get; set; }
    }
}

File: TransitoAplicacionMovil/app/src/main/java/mx/uv/lunix/transito/ws/pojos/Usuario.java
package mx.uv.lunix.transito.ws.pojos;

public class Usuario {

    private Integer IdUsuario;
    private String Nombre;
    private String ApellidoPaterno;
    private String ApelldoMaterno;
    private Integer IdCargo;
    private String Username;
    private String Password;

    public Integer getIdUsuario() {
        return IdUsuario;
    }

    public void setIdUsuario(Integer idUsuario) {
        IdUsuario = idUsuario;
    }

    public String getNombre() {
        return Nombre;
    }

    public void setNombre(String nombre) {
        Nombre = nombre;
    }

    public String getApellidoPaterno() {
        return ApellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        ApellidoPaterno = apellidoPaterno;
    }

    public String getApelldoMaterno() {
        return ApelldoMaterno;
    }

    public void setApelldoMaterno(String apelldoMaterno) {
        ApelldoMaterno = apelldoMaterno;
    }

    public Integer getIdCargo() {
        return IdCargo;
    }

    public void setIdCargo(Integer idCargo) {
        IdCargo = idCargo;
    }

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        Username = username;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }

    public Usuario(Integer idUsuario, String nombre, String apellidoPaterno, String apelldoMaterno, Integer idCargo, String username, String password) {
        IdUsuario = idUsuario;
        Nombre = nombre;
        ApellidoPaterno = apellidoPaterno;
        ApelldoMaterno = apelldoMaterno;
        IdCargo = idCargo;
        Username = username;
        Password = password;
    }
}
